website/client/components/inventory/equipment/attributesGrid.vue

```diff
@@ -0,0 +1,169 @@
+<template lang="pug">
+  div.attributes-group
+    .popover-content-attr(v-for="attr in ATTRIBUTES", :key="attr")
+      .group-content
+        span.popover-content-attr-cell.key(:class="{'hasValue': hasSumValue(attr) }") {{ `${$t(attr)}: ` }}
+        span.popover-content-attr-cell.label.value(:class="{'green': hasSumValue(attr) }") {{ `${stats.sum[attr]}` }}
+        span.popover-content-attr-cell.label.bold(:class="{'hasValue': hasGearValue(attr) }") Gear:
+        span.popover-content-attr-cell.label(:class="{'hasValue': hasGearValue(attr) }") {{ stats.gear[attr] }}
+        span.popover-content-attr-cell.label.bold(:class="{'hasValue': hasClassBonus(attr) }") Bonus:
+        span.popover-content-attr-cell.label(:class="{'hasValue': hasClassBonus(attr) }") {{ `${stats.classBonus[attr]}` }}
+</template>
+
+<style lang="scss">
+
+  @import '~client/assets/scss/colors.scss';
+
+
+  .attributes-group {
+    // unless we have a way to give a popover an id or class, it needs expand the attributes area
+    margin: -13px -16px;
+    display:flex;
+    flex-wrap: wrap;
+  }
+
+  .popover-content-attr {
+    font-weight: bold;
+    width: calc(50% - 1px);
+    background-color: #4e4a57;
+
+    &:nth-of-type(even) {
+      margin-left: 1px;
+    }
+
+    &:nth-child(1), &:nth-child(2) {
+      margin-bottom: 1px;
+    }
+  }
+
+  .group-content {
+    display: inline-flex;
+    flex-wrap: wrap;
+    padding: 4px 12px;
+    width: 100%;
+  }
+
+  .popover-content-attr-cell {
+    width: 50%;
+    text-align: left;
+
+    font-family: Roboto;
+
+    &:nth-of-type(even) {
+      text-align: right;
+    }
+
+    &.key {
+      color: $white;
+      font-size: 12px;
+      font-weight: bold;
+      line-height: 1.33;
+    }
+
+    &.label {
+      font-size: 10px;
+      line-height: 1.2;
+      color: #a5a1ac;
+    }
+
+    &.label.bold {
+      font-weight: bold;
+    }
+
+    &.label.value {
+      font-size: 12px;
+      font-weight: bold;
+      line-height: 1.33;
+      text-align: right;
+
+      &.green {
+        color: $green-10;
+
+        &:before {
+          content: '+';
+        }
+      }
+    }
+  }
+
+  .modal-body {
+
+    .group-content {
+      padding: 8px 17px;
+    }
+
+    .popover-content-attr {
+      background-color: #f4f4f4;
```

Comment:
color variable if possible

Reply:
none available, I could create one

Reply:
If it's not available I'd say if it's only used in one place let's not bother about using a var

Reply:
its also in `app.vue` but no where else I could extract both, any name suggestions? `$nearlyWhite` ? 😅

Reply:
not sure anymore I want a variable 😄